netbox/ipam/views.py

```diff
@@ -676,7 +676,7 @@ def get(self, request, pk):
             address=str(ipaddress.address)
         ).filter(
             vrf=ipaddress.vrf, address__net_contained_or_equal=str(ipaddress.address)
-        )
+        )[:50]
```

Comment:
Two requests:

1. Maybe use `settings.PAGINATE_COUNT` instead of hard-coding this value (it defaults to 50 anyway). This variable should be directly accessible from within the template.

2. Check whether the total number of records exceeds the displayed length. If so, include a link to the filtered IP addresses list for convenience.

Reply:
Done and done.